utils/line-directive

```diff
@@ -65,6 +65,8 @@ def fline_map(target_filename):
 def map_line_to_source_file(target_filename, target_line_num):
     """
     >>> from tempfile import *
+    >>> # We manually delete the temporary file as we can't open a temporary
+    >>> # on Windows due to platform differences.
```

Comment:
"due to platform differences" doesn't explain anything about why something has to be different on Windows, and "we can't open a temporary" doesn't make any sense, because that's what `NamedTemporaryFile` does.  I *think* what you mean to say is something like, "we can't open and read from a file that is already open for writing"